variables.tf

```diff
@@ -413,6 +413,12 @@ EXTENSIONS
   }
 }
 
+variable "guest_attestation_extension" {
+  type        = bool
+  default     = true
+  description = "(Optional) Set this value to true if the deployment should create a guest attestation extension for the virtual machine."
```

Comment:
Should we specify that this flag only works if `secure_boot_enabled` and `vtpm_enabled` are also set to true?